app/presenters/relationships_presenter.rb

```diff
@@ -0,0 +1,35 @@
+class RelationshipsPresenter
```

Comment:
Is it worth having a base presenter class, maybe inheriting from Delegator to forward attribute calls to the c100_application object?
https://ruby-doc.org/stdlib-2.4.0/libdoc/delegate/rdoc/Delegator.html

Reply:
Hmm, I don't think in this particular case would be worth it, as we are only accessing 2 methods from `c100_application` (#children and #relationships). But if we start creating other small presenters like this one, accessing more c100 methods/attributes, then it could be a good idea, yes.

Reply:
Yeah, that's what I meant. I think we have a couple of presenters that mostly delegate calls to the c100app object. If there's a pattern there, extracting a superclass might be sensible, and inheriting from delegator could simplify the code.